src/Tre-Hasura/HasuraQuery.cs

```diff
@@ -0,0 +1,140 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Hosting;
+using Newtonsoft.Json.Linq;
+using Microsoft.AspNetCore.Mvc.Routing;
+using Newtonsoft.Json;
+using Tre_Hasura.Models;
+using System.Net.Http.Headers;
```

Comment:
Potentially need to add `using System.Net.Http;` to allow a `HTTPClient object to be constructed.